docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -136,63 +201,17 @@ Here is an example role configuration snippet using the trait variable:
       '*': '*'
 ```
 
-## Step 3/3. Configure authentication preference
+## Step 3/4. Log in to Teleport using GitHub
 
-Configure the Teleport Auth Service to enable the GitHub authentication
-connector.
-
-Obtain your existing `cluster_auth_preference` resource:
-
-```code
-$ tctl get cap > cap.yaml
-```
-
-If you have not defined a `cluster_auth_preference`, `cap.yaml` will be blank.
-
-Ensure that `cap.yaml` includes the following content:
-
-```yaml
-kind: cluster_auth_preference
-metadata:
-  name: cluster-auth-preference
-spec:
-  type: github
-  webauthn:
-    rp_id: 'example.teleport.sh'
```

Comment:
@Tener while reviewing merge conflicts with `branch/v12` I realized that the configuration created by `tctl sso configure github` does not include `spec.webauthn.rp_id`. Is this something that the `tctl `command can include but is undocumented, and/or should we instruct the reader to manually add to `cap`?